diff --git a/dist/configResolved/plugins/resolveId.js b/dist/configResolved/plugins/resolveId.js
index d5c7927679b352e2e66bc9733c48276ece269ad6..0976d4a9164bc92abfcdb8791a0fa225bcc630e0 100644
--- a/dist/configResolved/plugins/resolveId.js
+++ b/dist/configResolved/plugins/resolveId.js
@@ -17,7 +17,7 @@ const BUILT_IN_MODULES = {
     '@dcloudio/uni-shared': 'dist/uni-shared.es.js',
     '@dcloudio/uni-stacktracey': 'dist/uni-stacktracey.es.js',
     '@vue/shared': 'dist/shared.esm-bundler.js',
-    pinia: 'dist/pinia.mjs',
+    pinia: '',
 };
 function uniResolveIdPlugin(options) {
     const resolveCache = {};
